README.md

```diff
@@ -129,11 +132,16 @@ In this case,
 }
 ```
 
+---
+**NOTE**
+
+This is also mandatory for users who want to extend the `mxGraph` integration.
+
+---
+
 Alternatively, you can set `skipLibCheck` to `true` in the `tsconfig.json` file, but this limits the definition checks.
 For more details, see the [skipLibCheck documentation](https://www.typescriptlang.org/tsconfig#skipLibCheck).
 
-Advanced users that want to extend the `mxGraph` integration must add `typed-mxgraph` to `typeRoots`.
-
 For more details, see the TypeScript projects in the [bpmn-visualization-examples repository](https://github.com/process-analytics/bpmn-visualization-examples/).
```

Comment:
now we have a repetition of "for more details" in the 2 sentences.

Reply:
```suggestion
```

As we have already have a section `For more details go to examples repo` bottom, I propose to delete it.

Reply:
rework the wording
change link to https://github.com/process-analytics/bpmn-visualization-examples/#bpmn-visualization-usage-in-projects for better guidance
Instead of "for more details" try with a information icon